### ISSUE TITLE
SCDM control  WF centers

### ISSUE THREAD

**Issue by Lijiuyin** [wannier-developers/wannier90#508]
Dear sir
I am using VASP + Wannier90 + TB2J to calculate magnetic interactions, and I found that the WF centers obtained using the SCDM method are not ideal and not around the atoms. I would like to know if this is because the SCDM method cannot control the WF centers, or if there are other parameters that can control them? 
my .win input file looks like：

'''guiding_centres= true
dis_num_iter  =  1000
num_iter      = 500
begin projections
Fe:s;p;d
Al:s;p;d
Te:s;p;d
end projections
dis_win_min   = -6.2
dis_win_max   = 32
dis_froz_min  = -6.2    # energy window
dis_froz_max  =  20
num_bands = 256
num_wann = 216
spinors = .true.

begin unit_cell_cart
......................
end unit_cell_cart
begin atoms_cart
......................
end atoms_cart
mp_grid =    10    10     1
#Bandstructure plot
#restart = plot
bands_plot = true
write_xyz = true
write_hr = true
begin kpoint_path
...............
end kpoint_path
begin kpoints
...............
end kpoints'''

Thank you very much for your attention and support.

Sincerely,
JiuYin Li

**Comment by matphy1** [comment]
May I ask if you have solved this problem? Could we have a chat?

**Comment by giovannipizzi** [comment]
Indeed, SCDM provides no guarantee on the location of the centers (and in most cases, especially for valence band only, they are *not* on atoms, often mid bond, but also this is not guaranteed
